### PR TITLE
Create coreWebAPI.md

### LINKED ISSUE #227: Provide include file for legacy Web API - consider ASP.NET Core Web API
I need to write an include file to add the top ASP.NET 4 Web API content that says consider ASP.NET Core Web API.

I'll add this to a bunch of ASP.NET 4 Web API including:

https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/action-results
https://docs.microsoft.com/en-us/aspnet/web-api/overview/formats-and-model-binding/parameter-binding-in-aspnet-web-api

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #227

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@danroth27  @scottaddie  we're getting alot of SEO hits to ASP.NET 4 Web API docs that folks really want ASP.NET Core Web API. I need to do something about this by Sept 13. This include is a start.

File: aspnet/includes/coreWebAPI.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Consider using [ASP.NET Core Web API](/aspnet/core/web-api). It has the follow advantages over ASP.NET 4 Web API:
```
Comment (scottaddie) [inline aspnet/includes/coreWebAPI.md]:
```suggestion
Consider using [ASP.NET Core web API](/aspnet/core/web-api). It has the following advantages over ASP.NET 4.x Web API:
```

File: aspnet/includes/coreWebAPI.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- ASP.NET Core is an open-source, cross-platform framework for building modern, cloud-based web apps on Windows, macOS, or Linux.
+- The ASP.NET Core MVC controller and Web API controller are unified.
```
Comment (scottaddie) [inline aspnet/includes/coreWebAPI.md]:
```suggestion
- The ASP.NET Core MVC controllers and web API controllers are unified.
```

File: aspnet/includes/coreWebAPI.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- Architected for testability.
+- Ability to develop and run on Windows, macOS, and Linux.
+- Open-source and community-focused.
+- Integration of modern, client-side frameworks and development workflows.
+- A cloud-ready, environment-based configuration system.
+- Built-in dependency injection.
+- A lightweight, high-performance, and modular HTTP request pipeline.
+- Ability to host on IIS, Nginx, Apache, Docker, or self-host in your own process.
```
Comment (scottaddie) [inline aspnet/includes/coreWebAPI.md]: Shouldn't Kestrel be listed here too now that it's suitable to be used as a public-facing edge server?

Comment (Rick-Anderson) [reply aspnet/includes/coreWebAPI.md]: See https://github.com/aspnet/AspNetCore.Docs/pull/14204

File: aspnet/includes/coreWebAPI.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Consider using [ASP.NET Core web API](/aspnet/core/web-api). It has the following advantages over ASP.NET 4.x Web API:
+
+- ASP.NET Core is an open-source, cross-platform framework for building modern, cloud-based web apps on Windows, macOS, or Linux.
```
Comment (guardrex) [inline aspnet/includes/coreWebAPI.md]:
```suggestion
* ASP.NET Core is an open-source, cross-platform framework for building modern, cloud-based web apps on Windows, macOS, and Linux.
```

File: aspnet/includes/coreWebAPI.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+- The ASP.NET Core MVC controllers and web API controllers are unified.
+- Architected for testability.
+- Ability to develop and run on Windows, macOS, and Linux.
+- Open-source and community-focused.
+- Integration of modern, client-side frameworks and development workflows.
+- A cloud-ready, environment-based configuration system.
+- Built-in dependency injection.
+- A lightweight, high-performance, and modular HTTP request pipeline.
+- Ability to host on [Kestrel](/aspnet/core/fundamentals/servers/kestrel), IIS, Nginx, Apache, or Docker.
```
Comment (guardrex) [inline aspnet/includes/coreWebAPI.md]:
```suggestion
- Ability to host on [Kestrel](/aspnet/core/fundamentals/servers/kestrel), [IIS](xref:host-and-deploy/iis/index, [HTTP.sys](xref:fundamentals/servers/httpsys), [Nginx](xref:host-and-deploy/linux-nginx), [Apache](xref:host-and-deploy/linux-apache), and [Docker](xref:host-and-deploy/docker/index).
```

**Comment by guardrex** [comment]
I put the rest of the markers in (*bored* ... waiting for 🌞 to go down for 🚴 ride).

**Pull request merged**